packages/widget-v2/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "widget-v2",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "storybook dev -p 6006",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "build-storybook": "storybook build"
```

Comment:
@codingki so basically dev would run the storybook dev

Reply:
yes but add it in our root package.json scripts. maybe dev:widgetv2 or something

Reply:
oh i see i misunderstood what you said